cfy_manager/components/postgresql_client/postgresql_client.py

```diff
@@ -92,34 +97,45 @@ def _create_postgres_user(self):
             else:
                 logger.info('User postgres already exists')
 
-    def _create_pgpass(self, host, port, db_name, user, password, pgpass_path,
+    def _create_pgpass(self, hosts, port, db_name, user, password, pgpass_path,
                        owning_user, owning_group):
         logger.debug('Creating postgresql pgpass file: {0}'
                      .format(pgpass_path))
-        pgpass_content = '{host}:{port}:{db_name}:{user}:{password}'.format(
-            host=host,
-            port=port,
-            db_name=db_name,
-            user=user,
-            password=password
-        )
+        line_template = '{host}:{port}:{db_name}:{user}:{password}\n'
+
+        pgpass_content = ''
+        for host in hosts:
+            pgpass_content = line_template.format(
```

Comment:
So pgpass_content would always have the information of the last host in the hosts' list. That's the behavior we're trying to get?

Reply:
...I didn't add it to the output <headdesk>